src/Uno.UI/UI/Xaml/DropUITarget.cs

```diff
@@ -0,0 +1,96 @@
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Windows.ApplicationModel.DataTransfer;
+using Windows.ApplicationModel.DataTransfer.DragDrop.Core;
+using Windows.Foundation;
+using Windows.UI.Input;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Input;
+
+namespace Windows.UI.Xaml
+{
+	internal class DropUITarget : ICoreDropOperationTarget
+	{
+		private readonly Window _window;
+
+		private readonly Dictionary<UIElement, (DraggingState state, DragUIOverride uiOverride, DataPackageOperation acceptedOperation)> _state
+			= new Dictionary<UIElement, (DraggingState state, DragUIOverride uiOverride, DataPackageOperation acceptedOperation)>();
+
+		public DropUITarget(Window window)
+		{
+			_window = window;
+		}
+
+		/// <inheritdoc />
+		public IAsyncOperation<DataPackageOperation> EnterAsync(CoreDragInfo dragInfo, CoreDragUIOverride dragUIOverride)
+			=> Task.FromResult(DataPackageOperation.Copy).AsAsyncOperation();
+
+		/// <inheritdoc />
+		public IAsyncOperation<DataPackageOperation> OverAsync(CoreDragInfo dragInfo, CoreDragUIOverride dragUIOverride)
+			=> Task.FromResult(DataPackageOperation.Link).AsAsyncOperation();
+		//{
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6270104)